### PR TITLE
docs: single-page view added

### PR DESCRIPTION
**Pull request by rahulyadav139**
demo: https://gifted-leakey-419629.netlify.app/

### CODE REVIEW

File: src/pages/ProductDetailPage.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  Footer,
+  SingleProductCard,
+  ProductDescription,
+} from '../components';
+import { Fragment } from 'react';
+
+const ProductDetailPage = props => {
+  return (
+    <Fragment>
+      <Header />
```
Comment (ivinayakg) [inline src/pages/ProductDetailPage.js:13]: add the header in the routes component that way its going to be rendered on every page

**Review by ivinayakg — APPROVED** [review]
try to set up the components for dynamic content, as that would be the end goal.

File: src/components/single-product-page/ProductDescription.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+          UV Sun protection - Protect your loved ones from UV rays, heat, bugs
+          and dust, all year around with noticeable heat reduction in cabin.
+        </li>
+        <li>
+          Covers all window surface - No sunlight leaks, unlike other car
+          shades. It is a total sun blocker with a double layer mesh, though
+          person seating on back seat can see through the window shade from the
+          inside out.
+        </li>
+        <li>
```
Comment (manojsarna) [inline src/components/single-product-page/ProductDescription.js:28]: Suggestion : Get this Data from Backend

File: src/components/single-product-page/SingleProductCard.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+        </h1>
+        <h2 class="product-title">Loud Speaker</h2>
+        <div class="rating">
+          4.4 <i class="fas fa-star"></i> | 3.9k Ratings
+        </div>
+        <div class="hr-line thin solid grey"></div>
+        <div class="price flex gap">
+          <div class="price__original">$100</div>
+          <div class="price__discounted">$80</div>
+        </div>
```
Comment (manojsarna) [inline src/components/single-product-page/SingleProductCard.js:25]: Suggestion : Get this data from backend

**Review by manojsarna — APPROVED** [review]
Do Check the comments

**Pull request merged**